### PR TITLE
Bug fix in adding expand and squeeze around 1d conv/pool

### PR DESCRIPTION
**Pull request by aseemw**
- another bug fix when end is -1 in slice layer.

### CODE REVIEW

**Comment by leonardoaraujosantos** [comment]
Hi Aseem, I tried at my side and it worked.
Screen Shot 2020-01-09 at 3 03 07 PM

File: onnx_coreml/_operators_nd.py [hunk 1]
```diff
@@ -1923,7 +1923,8 @@ def _convert_slice_ir4v9(builder, node, graph, err):
         current_axes = axes[i]
         starts[current_axes] = ip_starts[i]
         ends[current_axes] = ip_ends[i]
-        if ends[current_axes] != INT_MAX or ends[current_axes] < data_shape[current_axes]:
+        if ends[current_axes] < INT_MAX:
```
Comment (bhushan23) [inline onnx_coreml/_operators_nd.py]: Shouldn't we check with current data shape?

Comment (aseemw) [reply onnx_coreml/_operators_nd.py]: yeah that is something which I am not totally clear about. For example: if the data shape (i.e. a particular dimension) is 5 and end is 7, does that mean end is effectively -1 or 2

Comment (aseemw) [reply onnx_coreml/_operators_nd.py]: let me add test cases to cover all those cases, so that we know that code is correct

Comment (aseemw) [reply onnx_coreml/_operators_nd.py]: just checked the onnx documentation, if end value is larger than the dimension size, its treated as -1. Will update and add a test case.

Comment (aseemw) [reply onnx_coreml/_operators_nd.py]: updated it

File: tests/pytorch_model_test.py [hunk 2]
```diff
@@ -25,7 +26,7 @@
 
 MIN_MACOS_VERSION_10_15 = (10, 15)
 
-DEBUG = False
+DEBUG = True
```
Comment (bhushan23) [inline tests/pytorch_model_test.py]: Resetting to `False`

**Pull request merged**